seekers/tests.py

```diff
@@ -1,3 +1,67 @@
-# from django.test import TestCase
+from apartments.models import City
+from .forms import SeekerCreationForm
+from django.urls import reverse
+import pytest
 
-# Create your tests here.
+
+@pytest.fixture
+def new_city():
+    city = City(cityName='some_city')
+    city.save()
+    return city
+
+
+@pytest.mark.parametrize(
+    'city, start_date, min_rent, max_rent, num_of_roomates, num_of_rooms, about, validity',
+    [
+        ('new_city', '2021-2-2', 1000, 2000, 2, 2, 'about', True),
+        ('new_city', '', 1000, 2000, 2, 2, 'about', False),
+        ('new_city', '2021-2-2', None, 2000, 2, 2, 'about', False),
+        ('new_city', '2021-2-2', 1000, None, 2, 2, 'about', False),
+        ('new_city', '2021-2-2', 1000, 2000, None, 2, 'about', False),
+        ('new_city', '2021-2-2', 1000, 2000, 2, None, 'about', False),
+        ('new_city', '2021-2-2', 2000, 1000, 2, 2, 'about', False),
+        ('new_city', '2021-2-2', 1000, 2000, 2, 2, '', True),
+    ])
+@pytest.mark.django_db
+def test_seeker_form_validity(city, start_date, min_rent, max_rent, num_of_roomates, num_of_rooms, about,
+                              validity, request):
+
+    city = request.getfixturevalue(city)
+    form = SeekerCreationForm(data={
+        'city': city,
+        'start_date': start_date,
+        'min_rent': min_rent,
+        'max_rent': max_rent,
+        'num_of_roomates': num_of_roomates,
+        'num_of_rooms': num_of_rooms,
+        'about': about,
+    })
+
+    assert form.is_valid() is validity
+
+
+@pytest.fixture
+def valid_form(new_city):
+    return SeekerCreationForm(data={
```

Comment:
shouldn't you use `form.is_valid()`?

Reply:
Sorry, I didn't notice It's the fixture itself. 
Do not change it, but do add a separate test that asserts an insertion of a valid form was successful.

Reply:
Will do ☝️